pkg/operator/controller/ingress/status_test.go

```diff
@@ -182,16 +182,40 @@ func TestComputeLoadBalancerStatus(t *testing.T) {
 	}
 }
 
-func TestComputeIngressStatusConditions(t *testing.T) {
+func TestComputeIngressAvailableCondition(t *testing.T) {
 	testCases := []struct {
-		description     string
-		availRepl, repl int32
-		condType        string
-		condStatus      operatorv1.ConditionStatus
+		description string
+		deployment  []appsv1.DeploymentCondition
+		expect      operatorv1.OperatorCondition
```

Comment:
`expect` could be a Boolean value, with the `operatorv1.OperatorCondition` value still computed in the `for` loop.  Are you anticipating adding test cases that verify that the reason or message is accurately reflected?

Reply:
Maybe Reason. In any case, I just thought it was more clear to use the type directly in this case.